specification/baremetalinfrastructure/resource-manager/Microsoft.BareMetalInfrastructure/preview/2024-08-01-preview/baremetalinfrastructure.json

```diff
@@ -0,0 +1,1378 @@
+          },
+          {
+            "name": "requestBodyParameters",
+            "in": "body",
+            "description": "Resource create parameters.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/AzureBareMetalInstance"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/AzureBareMetalInstance"
+            }
+          },
+          "200": {
+            "description": "Azure operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/AzureBareMetalInstance"
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "AzureBareMetalInstances_Create": {
+            "$ref": "./examples/AzureBareMetalInstances_Create.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "Bare Metal Instances"
+        ],
+        "operationId": "AzureBareMetalInstances_Update",
+        "summary": "Patches the Tags field of a Azure Bare Metal Instance.",
+        "description": "Patches the Tags field of a Azure Bare Metal Instance for the specified subscription, resource group, and instance name.",
+        "x-ms-examples": {
+          "Update Tags field of an Azure Bare Metal Instance": {
+            "$ref": "./examples/AzureBareMetalInstances_PatchTags.json"
+          },
+          "Delete Tags field of an Azure Bare Metal Instance": {
+            "$ref": "./examples/AzureBareMetalInstances_PatchTags_Delete.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AzureBareMetalInstanceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/TagsParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/AzureBareMetalInstance"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
```

Comment:
Can you reference `v5` common types instead of `v3` common types for consistency?